spyder/plugins/ipythonconsole/widgets/figurebrowser.py

```diff
@@ -60,11 +60,11 @@ def _handle_display_data(self, msg):
             if (self.figurebrowser is not None and
                     self.figurebrowser.mute_inline_plotting):
                 if not self.sended_render_message:
-                    msg['content']['data']['text/plain'] = _(
-                        'Figures now render in the Plots pane by default. '
+                    msg['content']['data']['text/plain'] = self._append_html("<hr>"
```

Comment:
@juanis2112 please enclose all the message using the underscore function (`_`). This is the way we use to manage the translations of the strings (If you see the diff, before the changes, the string was enclosed in this function). Also, please use `'` to enclose the string (here the `<hr>` has `"` and the rest uses `'`)